spec/requests/v1/authentication_controller_spec.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe V1::AuthenticationController do
+end
```

Comment:
there is no controller with name Authentication controller

Reply:
done